Source/WebKit/UIProcess/DrawingAreaProxy.cpp

```diff
@@ -147,7 +147,7 @@ void DrawingAreaProxy::viewExposedRectChangedTimerFired()
     if (!m_webPageProxy->hasRunningProcess())
```

Comment:
We probably want to have a local Ref webPageProxy here to protect here and below.

Reply:
Fixed